src/NServiceBus.Core/DelayedDelivery/TimeoutManager/ExpiredTimeoutsPoller.cs

```diff
@@ -81,18 +83,13 @@ public void NewTimeoutRegistered(DateTime expiryTime)
             while (!cancellationToken.IsCancellationRequested)
             {
                 await SpinOnce().ConfigureAwait(false);
+                await Task.Delay(NextRetrievalPollSleep, cancellationToken).ConfigureAwait(false);
```

Comment:
You are right. I removed 

https://github.com/Particular/NServiceBus/commit/bd19aa53e647652a1aecc006d1a80c8e659aee28#diff-a579ea3ee7dd57a74feed0dc78afded5L90

which wasn't my attention. I just wanted to move the delay in the parent and leave the if there. I'll fix